sql/core/src/main/scala/org/apache/spark/sql/DataFrame.scala

```diff
@@ -1085,6 +1085,9 @@ class DataFrame private[sql](
    * there is no notion of a persisted catalog in a standard SQL context.  Instead you can write
    * an RDD out to a parquet file, and then register that file as a table.  This "table" can then
    * be the target of an `insertInto`.
+   *
+   * Also note that this doesn't create a hive table, but instead creates a Spark data source table.
```

Comment:
Maybe say

"Also note that while this function can persist the table metadata into Hive's metastore, the table will NOT be accessible from Hive."